internal/controller/deployment_to_pdb_controller.go

```diff
@@ -28,6 +29,9 @@ import (
 const PDBCreateAnnotationKey = "eviction-autoscaler.azure.com/pdb-create"
 const PDBCreateAnnotationFalse = "false"
 const PDBCreateAnnotationTrue = "true"
+const EnableEvictionAutoscalerAnnotationKey = "eviction-autoscaler.azure.com/enable"
+const KubeSystemNamespace = "kube-system"
```

Comment:
I learned only recently theres a k8s const for this. 

https://github.com/paulgmiller/nodesynack/commit/fcbdf940095490a3596232e1014debbfb7eb65d6

But see my other commetn that I think we should probably take this as an argument to contorller manager in opt in mode.